Anime-App/Feature/Anime/View/SettingsViewController.swift

```diff
@@ -0,0 +1,179 @@
+        animeImageView.image = UIImage(named: "gundam")
+        animeImageView.contentMode = .scaleAspectFit
+        animeImageView.translatesAutoresizingMaskIntoConstraints = false
+        return animeImageView
+    }()
+    
+    private lazy var hapticFeedBackStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        stackView.spacing = 32
+        return stackView
+    }()
+    
+    private lazy var audioStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        stackView.spacing = 32
+        return stackView
+    }()
+    
+    private lazy var appVersionLbl: UILabel = {
+        let appVersionLbl = UILabel()
+        appVersionLbl.translatesAutoresizingMaskIntoConstraints = false
+        appVersionLbl.textColor = .black
+        return appVersionLbl
+    }()
+    
+    private lazy var hapticFeedbackLbl: UILabel = {
+        let hapticFeedbackLbl = UILabel()
+        hapticFeedbackLbl.translatesAutoresizingMaskIntoConstraints = false
+        hapticFeedbackLbl.textColor = .black
+        hapticFeedbackLbl.text =  "haptic"
+        hapticFeedbackLbl.font = .init(name: "", size: 14)
+        return hapticFeedbackLbl
+    }()
+    
+    
+    private lazy var audioLbl: UILabel = {
+        let audioLbl = UILabel()
+        audioLbl.translatesAutoresizingMaskIntoConstraints = false
+        audioLbl.textColor = .black
+        audioLbl.text =  "Audio"
+        audioLbl.font = .init(name: "", size: 14)
+        return audioLbl
+    }()
+    
+    private lazy var hapticFeedbackSwitch: UISwitch = {
+        let hapticFeedbackSwitch = UISwitch()
+        hapticFeedbackSwitch.translatesAutoresizingMaskIntoConstraints = false
+        hapticFeedbackSwitch.isOn = false
+        hapticFeedbackSwitch.addTarget(self, action: #selector(toggleHapticFeedback), for: .touchUpInside)
+        return hapticFeedbackSwitch
+    }()
+    
+    private lazy var audioSwitch: UISwitch = {
+        let audioSwitch = UISwitch()
+        audioSwitch.addTarget(self, action: #selector(toggleAudio), for: .touchUpInside)
+        audioSwitch.translatesAutoresizingMaskIntoConstraints = false
+        return audioSwitch
+    }()
+    
+    @objc func toggleHapticFeedback() {
+        if hapticFeedbackSwitch.isOn == true {
+            print("haptic Feedback is on")
+            defaults.set(true, forKey: "UseHaptic")
+        } else {
+            UserDefaults.standard.set(false, forKey: "UseHaptic")
+            guard let appDomain = Bundle.main.bundleIdentifier else { return }
+            UserDefaults.standard.removePersistentDomain(forName: appDomain)
```

Comment:
So you don't want to do this where you remove it from user defaults you just want to set the the flag in user defaults to true or false. Also you could do it in one like this too.

```

 defaults.set(hapticFeedbackSwitch.isOn, forKey: "UseHaptic")

```